shap/plots/_image.py

```diff
@@ -127,3 +132,381 @@ def image(shap_values, pixel_values=None, labels=None, width=20, aspect=0.2, hsp
     cb.outline.set_visible(False)
     if show:
         pl.show()
+
+
+def image_to_text(shap_values):
+
+    from IPython.core.display import display, HTML
+
+    if len(shap_values.values.shape) == 5:
+        for i in range(shap_values.values.shape[0]):
+            display(HTML("<br/><b>"+ordinal_str(i)+" instance:</b><br/>"))
+            image_to_text(shap_values[i])
+        
+        return
+    
+
+    uuid = ''.join(random.choices(string.ascii_lowercase, k=20))
+    
+    # creating input html tokens
+    
+
+    model_output = shap_values.output_names
+
+    output_text_html = ''
+
+    for i in range(model_output.shape[0]):
+        output_text_html += "<div style='display:inline; text-align:center;'>" \
```

Comment:
nit: hmm I don't think you need the \ at end of line if you are using + operator (?)

Reply:
I think its required, otherwise the next line gets an unexpected indentation error